pandas/formats/format.py

```diff
@@ -1902,11 +1907,26 @@ def _format_hierarchical_rows(self):
                                         indexcolval, header_style)
                     gcolidx += 1
 
+        for cell in self._generate_body(coloffset=gcolidx):
```

Comment:
And if uninterested in merging the whole patch, could you consider merging just this refactoring, so that it's easier to extend `ExcelFormatter`?